core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -688,4 +688,65 @@ package object config {
       .stringConf
       .toSequence
       .createWithDefault(Nil)
+
+  private[spark] val KAFKA_BOOTSTRAP_SERVERS =
```

Comment:
This pattern was added after you PR, but since it's there... might be better to put these into a new file (e.g. `Kafka.scala`) in this package, instead of adding more stuff to this file. (e.g. see `History.scala`)

Reply:
Moved.